src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -501,97 +451,11 @@ int LuaFloatingWindowIsVR(lua_State *L) {
 
     FloatingWindow *wnd = (FloatingWindow *) lua_touserdata(L, 1);
 
-    lua_pushnumber(L, wnd->isVR());
+    lua_pushboolean(L, wnd->isVR());
```

Comment:
LuaSetFloatingWindowGeometry does not seem to set the window geometry. Instead, it returns whether the window is in VR. This doesn't seem intentional.

Reply:
I don't see this, looking at the code via GitHub, line 342, it is setting geometry.

Reply:
And for your other comments that I can't seem to reply to:

LuaSetFloatingWindowPosition: I fixed by checking all 3 parms, also changed the check of the 2nd parm from isstring to isnumber

Fixed the indentation, not sure what happened there.

For the one set_geometry function, I can do that, but I need to know how VR windows work, can they be both floating and popped? Will also have to check the is_popped flag.
The other thing with set_geometry is that VR only takes 2 parms, width & height, whereas the other 2 functions take 4 parms. I could just ignore parms 3 & 4 if passed when wanting to change VR geometry.